internal/controller/api/v1beta1/agentserviceconfig_types.go

```diff
@@ -22,6 +22,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// OSImage defines an Operating System image and the OpenShift version it
+// is associated with.
+type OSImage struct {
+	// OpenshiftVersion is the Major.Minor version of OpenShift that this image
+	// is to be associated with.
+	OpenshiftVersion string `json:"openshiftVersion"`
+	// Version is the Operating System version of the image.
```

Comment:
It's probably worth specifying that it should be `X.Y` only.

Reply:
That's not the case for the OS, right? It says that above for the openshift version.

Reply:
Ah yes, thanks! Nothing to see here... :)